include/alpaka/mem/buf/BufCudaRt.hpp

```diff
@@ -436,8 +435,7 @@ namespace alpaka
                                 &pitchBytes,
                                 widthBytes,
                                 height));
-                        assert(memPtr);
-                        assert(pitchBytes>=widthBytes);
+                        assert(pitchBytes>=widthBytes||(width*height)==0);
```

Comment:
I am not sure. If the with is zero the old assert is enough but what is if `height` is zero. Therefore I added the second check.